derive_builder/tests/compile-fail/build_fn_error.stderr

```diff
@@ -1,31 +1,23 @@
-error: Unknown field: `path`
+error: unrecognized derive_builder attribute
  --> tests/compile-fail/build_fn_error.rs:4:52
   |
 4 | #[builder(build_fn(error(validation_error = false, path = "hello")))]
   |                                                    ^^^^
 
-error: Cannot set `error(validation_error = false)` when using `validate`
-  --> tests/compile-fail/build_fn_error.rs:10:45
+error: `error(validation_error = false)` cannot be set when using `validate`
```

Comment:
if `validate = "hello"` is highlighted below then shouldn't this say something like "`validate` cannot be set if `error(validation_error = false)` is used"? :)

Reply:
Good call. I updated the PR rewording it to "**`error(validation_error = false)` and `validate` cannot be used together**" which is consistent with wording that is already used prior to this PR when dealing with the conflict between `builder(default)` and `builder(field(build = "..."))`.